presto-main/src/test/java/io/prestosql/sql/planner/TestLocalDynamicFilterCollector.java

```diff
@@ -27,7 +27,7 @@
 import static io.prestosql.spi.type.BigintType.BIGINT;
 import static io.prestosql.testing.assertions.Assert.assertEquals;
 
-public class TestLocalDynamicFiltersCollector
+public class TestLocalDynamicFilterCollector
```

Comment:
nit: this should be separate commit

Reply:
Rename was unintentional, I've reverted it.